skimage/__init__.py

```diff
@@ -68,14 +68,15 @@
 __version__ = '0.15dev'
 
 
-if sys.version_info < (3,):
+if sys.version_info < (3, 5):
```

Comment:
JFYI, there was a discussion on the best approach for version comparison - https://github.com/scikit-image/scikit-image/pull/2975

Reply:
@soupault I think that discussion is more relevant to third-party packages where the version string is less predictable than Python system versions